dev-util/lockrun/files/lockrun.c-20240415

```diff
@@ -0,0 +1,417 @@
+/*
```

Comment:
No, please don't. I realize that the current version is doing this, but we really ought to stop hosting code in FILESDIR. Just put it anywhere and use SRC_URI.